### DISCUSSION TITLE
Chit-chat

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#344] [General]
Wanted to share a link I just found on Bsky:
https://www.beamrad.io/81

Where can we share stuff like this and casually chat?

**Comment by wintermeyer** [comment]
We should create a wiki page for these kind of resources and add them there.

**Comment by fradarr** [comment]
Let's go for the wiki so we can see if the "chating" function in form of comments and edits fits us good enough. By the way, we need is a chat in Omedis anyway!
